module/merintr/inventry.c

```diff
@@ -671,15 +672,28 @@ void InventoryLButtonDown(HWND hwnd, BOOL fDoubleClick, int x, int y, UINT keyFl
 /************************************************************************/
 /*
  * InventoryLButtonUp:  User released left mouse button on inventory box.
+ * Call InventoryMoveCurrentItem if the mouse is within the inventory.
  */
 void InventoryLButtonUp(HWND hwnd, int x, int y, UINT keyFlags)
 {
    int temp_x, temp_y;
    room_contents_node *r;
+   POINT mouse;
+   AREA a;
+
+   InventoryGetArea(&a);
+   GetCursorPos(&mouse);
+   ScreenToClient(cinfo->hMain, &mouse);
 
    if (!InventoryReleaseCapture())
       return;
 
+   if (IsInArea(&a, mouse.x, mouse.y))
```

Comment:
But this does need a comment, e.g. "If button was released in inventory area, move selected item to new location in inventory list"

Reply:
Added comment as requested.